package.json

```diff
@@ -66,6 +66,8 @@
         "bootstrap-select": "^1.13.16",
         "bootstrap-vue": "^2.15.0",
         "es6-promise": "^4.2.8",
+        "laravel-mix-merge-manifest": "^0.1.2",
+        "moment": "^2.27.0",
```

Comment:
We shouldn't add this library any more: https://www.golem.de/news/moment-js-entwicklung-millionenfach-genutzter-js-bibliothek-beendet-2009-150903.html

Reply:
https://github.com/THM-Health/PILOS/pull/23/commits/1a33a7aa89dddde61afd7c0994421ca8d1e20bfb removed